src/protos/io/defang/v1/fabric.proto

```diff
@@ -690,3 +693,25 @@ message WhoAmIResponse {
   SubscriptionTier tier = 5;
   bool training_opt_out = 6;
 }
+
+message EstimateRequest {
+  string provider = 1;
```

Comment:
This is an enum.

Reply:
I don't understand the suggestion

Reply:
Provider is an enum `ProviderID`

Reply:
Oh you're suggesting that I use an enum instead of a string. OK. Can do.